README.md

```diff
@@ -34,6 +34,238 @@ To be notified whenever an MTD API change is deployed to the Sandbox or Producti
+
+* Some error response status codes are corrected from 403 to 400 for the following endpoints:
+  * Retrieve Income Tax (Self Assessment) Crystallisation Obligations
+  * Retrieve Income Tax (Self Assessment) End of Period Statement Obligations
+  * Retrieve Income Tax (Self Assessment) Income and Expenditure Obligations
+* Updated documentation:
+  * Updated the description for `ruleFromDateNotSupported` error
+  * Renamed endpoint from `Retrieve Income Tax (Self Assessment) Crystallisation Obligations` to `Retrieve Income Tax (Self Assessment) Final Declaration Obligations`
+  * Updated the downstream URL for all three endpoints
+
+#### other-deductions-api
+
+The following changes are now available in production:
+
+* Updated the description and message for `RULE_TAX_YEAR_NOT_SUPPORTED` to indicate that there could be a maximum as well as a minimum supported tax year
+* Updated documentation
+* Endpoint `Amend Other Deductions` renamed to `Create And Amend Other Deductions`
+
+#### property-business-api
+
+The following changes are now available in production:
+
+* Updated the descriptions of:
+  * `Amend a Historic FHL UK Property Income & Expenses Period Summary`
+  * `Create a Historic FHL UK Property Income & Expenses Period Summary`
+  * `Create a Historic Non-FHL UK Property Income & Expenses Period Summary`
+
+#### self-assessment-accounts-api
+
+The following changes are now available in production:
+* Updated `List Self Assessment Payments & Allocation Details` - added `MISSING_PAYMENT_LOT_ITEM` and `RULE_INCONSISTENT_QUERY_PARAMS` errors
+
+* Updated documentation:
+  * Updated the description of `List Self Assessment Payments & Allocation Details`
+  * Updated the description and message for `RULE_TAX_YEAR_NOT_SUPPORTED` to indicate that there could be a maximum as well as a minimum supported tax year
+
+#### self-assessment-biss-api
+The following changes are now available in production:
+
+* Error response status code corrected from 403 to 400 for `Retrieve a Business Income Source Summary`
+
+#### self-assessment-bsas-api
+The following changes are now available in production:
+
+* Some error response status codes are corrected from 403 to 400 for the `v3.0` following endpoints:
+  * Submit Foreign Property Accounting Adjustments
+  * Submit Self-Employment Accounting Adjustments
+  * Submit UK Property Accounting Adjustments
+  * Trigger a Business Source Adjustable Summary
+
+#### self-employment-business-api
+The following changes are now available in production:
+
+* New API Version `v2.0`. The following endpoints have been updated to replace their respective v1 equivalents. Please use the new v2 endpoints instead.
+
+* `Create a Self-Employment Period Summary`
+  * Added `RULE_INVALID_SUBMISSION_PERIOD` and `RULE_INVALID_SUBMISSION_END_DATE` errors
+  * Renamed `periodAllowableExpenses` and all its properties, removing 'allowable', for example `periodAllowableExpenses` becomes `periodExpenses`
```

Comment:
To avoid the repetition, could these 3 be reworded as:

```
* `Create a Self-Employment Period Summary`
  * Added `RULE_INVALID_SUBMISSION_PERIOD` and `RULE_INVALID_SUBMISSION_END_DATE` errors.

* `Create a Self-Employment Period Summary`, `Retrieve a Self-Employment Period Summary`, `Amend a Self-Employment Period Summary`
  * In all 3 endpoints, renamed `periodAllowableExpenses` and all its properties to remove the word "allowable"; e.g. `periodAllowableExpenses` becomes `periodExpenses`.
 ```